espresso/environment/6_batch_inbox_test.go

```diff
@@ -0,0 +1,159 @@
+package environment_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	env "github.com/ethereum-optimism/optimism/espresso/environment"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/geth"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/setuputils"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr/metrics"
+	ethereum "github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/stretchr/testify/require"
+)
+
+const MOCK_AUTHENTICATOR_ADDR = "0xBAD8ae8e955D24D64D0b21b839ceF63075757Bc7"
+
+// TestE2eDevNetWithInvalidAttestation verifies that the batcher correctly fails to register
```

Comment:
Looks like this comment belongs to test 5.

Reply:
Oops, I copy-pasted it as a template and forgot to change the comment